src/commands/createNewProject/validateFunctionProjects.ts

```diff
@@ -68,3 +75,67 @@ function isInitializedProject(folderPath: string): boolean {
     const runtime: string | undefined = getFuncExtensionSetting(projectRuntimeSetting, folderPath);
     return !!language && !!runtime;
 }
+
+/**
+ * JavaScript debugging in the func cli had breaking changes in v2.0.1-beta.30. This verifies users are up-to-date with the latest working debug config.
+ * See https://aka.ms/AA1vrxa for more info
+ */
+async function verifyDebugConfigIsValid(folderPath: string, actionContext: IActionContext): Promise<void> {
```

Comment:
Need to guard against this function throwing?

Reply:
good point